.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -21,3 +21,9 @@ Did you install the stable istio.yaml, istio-auth.yaml.... or if using the Helm
 
 **Environment**
 Which environment, cloud vendor, OS, etc are you using?
+
+**Cluster state**
+Follow
+[instructions](http://preliminary.istio.io/help/bugs/#generating-a-cluster-state-archive)
+to generate "istio-dump.tar.gz", then attach it here by dragging and dropping
+the file onto this issue.
```

Comment:
really? This will make every issue huge.. Its also a lot of info. Unless its a bug that someone is actively debugging, 99% of time, this will just be bloat.

Reply:
We shouldn't point the user to preliminary.istio.io, should be istio.io. We should also preface this with:

If you're running on Kubernetes, consider following the instructions...

@rshriram Does the size impact anything in the real-world? We're not paying for storage, so does it matter?

Reply:
@geeknoid I wasn't sure what link to put there since istio/istio.github.io#1892 will only reveal it on the preliminary site, and that link would be broken.. on second thought, this PR is merged into release-1.0 so the timeline will line up. I also like your preface so I added that. Done.